cmd/prober/prober.go

```diff
@@ -185,7 +194,18 @@ func main() {
 	Logger.Fatal(http.ListenAndServe(addr, nil))
 }
 
-func runProbers(ctx context.Context, freq int, runOnce bool) {
+func NewFulcioClient() (fulciopb.CAClient, error) {
```

Comment:
nit, I would mention "GRPC" in the name since this is only creating a gRPC client. Also you can make this method lowercase so it's not exported.